### PR TITLE
[INT-929] Added new PluginMethod of Exporter (getSelectedAutoExportParams)

### PR DESCRIPTION
**Pull request by agataHappy**
**Ticket**
https://dronedeploy.atlassian.net/browse/INT-929

**Work done**
Added new plugin method of Exporter to docs with example of usage.

**Screenshots**

Zrzut ekranu z 2019-05-31 09-31-38

Zrzut ekranu z 2019-05-31 09-20-36

### CODE REVIEW

**Comment by mikolaj-leszczynski** [comment]
@agataHappy can you attach screenshots which show the work you have done?

File: docs/exporter/example-exporter.getselectedautoexportparams.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        }
+        .exportValue {
+          font-family: Lato, sans-serif;
+          color: rgba(0, 0, 0, 0.87);
+        }
+    </style>
+</head>
+
+<body>
+    <h1 class="title sans">Export.send example</h1>
```
Comment (mikolaj-leszczynski) [inline docs/exporter/example-exporter.getselectedautoexportparams.md]: Ooops, something wen wrong when copying :P

File: docs/exporter/example-exporter.getselectedautoexportparams.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+    <button type="button" id="export" class="btn">Export</button>
```
Comment (mikolaj-leszczynski) [inline docs/exporter/example-exporter.getselectedautoexportparams.md]: and here

**Review by mikolaj-leszczynski — CHANGES_REQUESTED** [review]

File: docs/exporter/example-exporter.getselectedautoexportparams.md (hunk 1 above)
Comment (agataHappy) [reply docs/exporter/example-exporter.getselectedautoexportparams.md]: Done

File: docs/exporter/example-exporter.getselectedautoexportparams.md (hunk 2 above)
Comment (agataHappy) [reply docs/exporter/example-exporter.getselectedautoexportparams.md]: Done

**Review by mikolaj-leszczynski — APPROVED** [review]

**Pull request merged**